tests/integration/components/lt-body-test.js

```diff
@@ -57,16 +57,40 @@ test('row selection', function(assert) {
   assert.equal(this.$('tbody > tr').length, 5);
 
   firstRow.click();
-  assert.equal(this.$('tr.is-selected').length, 1);
+  assert.equal(this.$('tr.is-selected').length, 1, 'clicking a row selects it');
 
   lastRow.trigger(createClickEvent({shiftKey: true}));
-  assert.equal(this.$('tr.is-selected').length, 5);
+  assert.equal(this.$('tr.is-selected').length, 5, 'shift-clicking another row selects it and all rows between');
 
   middleRow.trigger(createClickEvent({ctrlKey: true}));
-  assert.equal(this.$('tr.is-selected').length, 4);
+  assert.equal(this.$('tr.is-selected').length, 4, 'ctrl-clicking a selected row deselects it');
 
   firstRow.click();
-  assert.equal(this.$('tr.is-selected').length, 0);
+  assert.equal(this.$('tr.is-selected').length, 0, 'clicking a selected row deselects all rows');
+});
+
+test('row selection - click to modify selection', function(assert) {
```

Comment:
this new test is very much based on the standard mode test before it, exercising and documenting the differences in behavior for (nearly) identical user input